src/clientsettingsdlg.cpp

```diff
@@ -823,7 +823,8 @@ void CClientSettingsDlg::UpdateSoundCardFrame()
     else
     {
         // special title text with buffer size information added
-        grbSoundCrdBufDelay->setTitle ( tr ( "Buffer Delay: " ) + GenSndCrdBufferDelayString ( iCurActualBufSize ) );
+        // append 2 whitespace characters, so that the text doesn't get cut off
+        grbSoundCrdBufDelay->setTitle ( tr ( "Buffer Delay: " ) + GenSndCrdBufferDelayString ( iCurActualBufSize ) + "  " );
```

Comment:
I am currently using Ubuntu 20.10 with Gnome 40.4.0 and Wayland. Did you test on Ubuntu? It might depend on the UI font, which is "Ubuntu Regular, 11" for me (you can set that with gnome-tweaks)

Reply:
(Point of order:

When a PR is fixing a defect it should provide
- description of the defect, including how to reproduce it and, if it's a UI defect, screenshots
- description of the fix and, if it's a UI defect, screenshots

along with everything else.)

Reply:
The PR should include "after" screenshots if they're not included in the issue (which they're not here).